pom.xml

```diff
@@ -98,19 +98,19 @@
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-api</artifactId>
-            <version>5.4.0-M1</version>
+            <version>5.4.2</version>
```

Comment:
Funker ting med 5.5.0-M1? Hvis så, like greit å bare gå for latest greatest? Test-avhengighet, så milestone-release går vel greit.

I tillegg, hvis du gidder, sleng gjerne inn `dependencyManagement`-seksjon med import-scope avhengighet til [JUnit BOM-en](https://search.maven.org/artifact/org.junit/junit-bom/5.5.0-M1/pom), så slipper du å spekke opp alle disse versjonene til ulike JUnit-artefakter

Reply:
Du er så avantgarde! :)

Reply:
For meg kunne gjerne Ultima-festivalen vært året rundt! ;)

Reply:
df46283